docs/website/tds/reference/NetcdfSubsetServiceConfigure.html

```diff
@@ -48,14 +48,14 @@ <h2><a name="restrictions">Restrictions on what files can be served</a></h2>
 
 <p>
     First, only datasets in the format that the CDM can read are supported. Second, the data must represent one of the
-    following <a href="/software/netcdf-java/v4.6/reference/FeatureDatasets/Overview.html">Feature Types</a>: GRID, POINT,
+    following <a href="/netcdf-java/4.6/userguide/reference/FeatureDatasets/Overview.html">Feature Types</a>: GRID, POINT,
     STATION.
 </p>
 
 <h3>Verifying that a dataset is gridded</h3>
 
 <p>Open your file in the
-    <a href="https://www.unidata.ucar.edu/software/netcdf-java/v4.6/webstart/netCDFtools.jnlp">NetCDF ToolsUI
+    <a href="https://www.unidata.ucar.edu/netcdf-java/4.6/userguide/webstart/netCDFtools.jnlp">NetCDF ToolsUI
```

Comment:
Let's just reference the URL on the docs server directly and not rely on the redirect we will put into place.